### DISCUSSION TITLE
Spotify Premium Duo, not qualified for Premium?

### DISCUSSION THREAD

**Discussion by Azuria** [music-assistant/support#973] [Q&A]
Hi
I recently learned about MA in a Youtube video, so of course I jumped on :D

But when I add in my Spotify credentials I get the following error: **Setup failed of provider spotify: Only Spotify Premium accounts are supported**

I have Premium Duo, is Duo not supported or what am I doing wrong?

Kind regards

**Comment by OzGav** [comment]
Good question. That should be supported. Let’s see what @marcelveldt says.

**Comment by OzGav** [comment]
Relates to v1
